src/less/mixins/carousel/ds6/carousel-mixins.less

```diff
@@ -6,6 +6,7 @@
 
     border-radius: 16px;
     box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.24), 0 0 2px 0 rgba(0, 0, 0, 0.12);
+    display: flex;
```

Comment:
A cleaner solution would be to add 
`justify-content: center;` next to `display:flex`
And then the `flex: 1` would no longer be needed (along with all those selectors)

Reply:
For some reason I added the `flex` thinking that `justify-content` didn't work, but alas, it does. Thanks for the suggestion to try it again.